lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenProjectCache.java

```diff
@@ -167,24 +165,25 @@ public InputStream getInputStream() throws IOException {
 					File workingCopy = null;
 					try {
```

Comment:
you should get rid of this try(...) by the way.

Reply:
Done and also I clean up the remnant workingCopy file reference as well.